docs/notifications.md

```diff
@@ -531,6 +537,37 @@ Example:
 }
 ```
 
+### `header_of_added_block` notification
+Notification is sent when new block is stored, which is happens after block's processing.
+
+Example:
+
+```
+{
+   "jsonrpc": "2.0",
+   "method": "header_of_added_block",
+   "params": [
+      {
+         "hash": "0x5d39141665a61c70d2ddbb997326e86e4b7f5eb34be43c7cd7823c4be513a71b",
+         "index": 8,
+         "merkleroot": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Take as an example the block mentioned in `block_added`, so that we have all header fields set.